docs/index.html

```diff
@@ -420,7 +420,7 @@ <h4 class="code-container-style"><b>View stats of your hosted challenge with ID
                 </div>
                 <p id="footer-margin">
                     Maintained by the EvalAI core team
-                    <br> ©2019 EvalAI
+                    &copy;<script>new Date().getFullYear()&&document.write(new Date().getFullYear());</script> EvalA
```

Comment:
Add `br` for new line.

Reply:
I think there is no need to add the <br> tag. I have tried adding it, it makes no change in the display

Reply:
By adding `br` tag, `©2019 EvalAI` will come on the next line!

Reply:
It's anyway coming on the next line. I'll change it if you say so